index.html

```diff
@@ -71,7 +71,9 @@
 	<li class="delete separator"><a href="#delete"></a></li>
 </ul>
 
-<script type="text/javascript" src="scripts/jquery-1.8.3.min.js"></script>
+<script type="text/javascript" src="vendor/compoents/jquery/jquery.min.js"></script>
```

Comment:
Yes, the new jQuery version should be in scripts folder and implicitly called from index.html. I know some of you, guys, are waiting for a [package manager version](https://github.com/simogeo/Filemanager/issues/293) (and I understand).